src/_rules/border.js

```diff
@@ -18,6 +18,7 @@ const borderStyles = [
 export const borders = [
   [/^border$/, handlerBorder],
   [/^border-transparent$/, () => ({ 'border-color': 'transparent' })],
+  [/^border-white$/, () => ({ 'border-color': '#fff' })],
```

Comment:
`border-white`: Used in Finn, and not in Warp yet. It's from [tailwind](https://v1.tailwindcss.com/docs/border-color)

Reply:
I think we want to use `var(--w-white)` here instead of a hex value

Reply:
Are we sure that we really want a `border-white` class though? 🤷  It says in the missing classes sheets that it is not used in Finn? And if we want to enable dark-mode, we would probably want whatever element this is set on to have a black border instead? We have `inverted` semantic tokens for background, icon and text... so we might want that for border as well? (ping @adidick)

Reply:
`border-white` is used a lot in Finn. I found a lot of usages. Good point about the inverted tokens. I guess we should do the same for borders, let's see what Adi answers

Reply:
Yes, I searched in the wrong place, so it is used a lot. I'll change it to `var(--w-white)` at least :) Thanks!

Reply:
I really don't think that we should add this. Simply because it will break dark-mode (if that is something we'll be doing?). Or... if we add a brand that has a dark background as default, not having a semantic class will give us no chance to override this for that brand... white is white. 🤷  I talked to Adi about this, and I think we should add a `border-inverted` semantic token... so then we can point any usage of `border-white` to instead use `s-border-inverted`. Wdyt?

Reply:
Semantic tokens for inverted border (`s-border-inverted`) added in this PR : https://github.com/warp-ds/css/pull/22

Reply:
Good idea! I changed it now @magnuh!